ParallelWithTestNG/src/main/java/WebTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Created by Nghi Thai on 12/29/17.
+ */
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import org.testng.ITestContext;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import java.util.Map;
+
+public class WebTest {
+    public static Map<String, String> testngParams;
+    String kobitonURL;
+    RemoteWebDriver driver = null;
```

Comment:
I saw in our sample using `RemoteWebDriver` (It's too generic). 
* Refer to use as Appium sample code: `AppiumDriver driver = null;`
* Refactor other classes too.